nsd.h

```diff
@@ -26,6 +26,7 @@ struct daemon_remote;
 #ifdef USE_DNSTAP
 struct dt_collector;
 #endif
+struct nsd_verifier_proxy;
 
```

Comment:
It looks like the `struct nsd_verifier_proxy` is not used. If so, you can delete this line.

Reply:
A remnant from a previous work. Thanks for catching this.